app/controllers/admins/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Admins::SessionsController < Devise::SessionsController
```

Comment:
We need to have class Admins::SessionsController and this is where the generators put it and the contents are still the default from the generator. I could get rid of the comments if you want ?